microprofile/reactive-streams-operators-smallrye/cdi-provider/src/main/java/org/wildfly/extension/microprofile/reactive/streams/operators/cdi/ReactiveEngineProvider.java

```diff
@@ -35,6 +35,7 @@
 /**
  * @author <a href="mailto:kabir.khan@jboss.com">Kabir Khan</a>
  */
+@ApplicationScoped
```

Comment:
It is correct to place a bean defining annotation here but since I don't know the internals of its usage, I can't say whether `@Dependent` or `@ApplicationScoped`.
Though I can tell you that it was previously (with discovery mode `all`) considered a `@Dependent` bean.

Reply:
Done -thanks for the feedback!